.qbraid.yaml

```diff
@@ -1,14 +1,14 @@
 # Sample qBraid environment configuration
-name: "example-action-environment"
-description: "Environment for testing environment publish action."
+name: "My Environment"
+description: "Example qBraid environment for demonstration purposes"
 tags:
   - "qbraid"
   - "environment"
-  - "core"
-  - "action"
+  - "example"
 icon: "icon.png"
 python_version: "3.11.6"
-kernel_name: "Python 3[qbraid-action-test]"
-shell_prompt: "qbraid_action_test"
+kernel_name: "Python 3 [MyEnv]"
+shell_prompt: "my-env"
 python_packages:
-  qbraid-core: ">=0.1.36"
\ No newline at end of file
+  pyqasm: ">=0.3.0"
+  matplotlib: ""
```

Comment:
Any particular reason for the package changes?

Reply:
no reason in particular, just showing two different package syntaxes, one with a version specified and one without. pyqasm + matplotlib seemed like a good combo